src/features/counterfactual/PayNowPayLater.tsx

```diff
@@ -42,6 +45,12 @@ const PayNowPayLater = ({
       <Typography variant="h4" fontWeight="bold">
         Before you continue
       </Typography>
+      {isMultiChain && (
+        <ErrorMessage level="info">
+          You will need to <b>activate your account</b> separately, on each network. Make sure you have funds on your
```

Comment:
I feel like "Make sure you have funds on your wallet..." sounds like the user needs to have funds to continue. I might drop that sentence as the user only needs funds at activation time.

Reply:
This component still renders the text "If you choose to pay later, the fee will be included with the first transaction you make." for multichain Safes.